example-app/src/app.module.ts

```diff
@@ -1,8 +1,7 @@
 import { Module } from '@nestjs/common';
 import { MongooseModule, getModelToken } from '@nestjs/mongoose';
-import mongoose, { Model } from 'mongoose';
-
-import { AdminModule } from '../../src/index';
+import { AdminModule } from '@admin-bro/nestjs';
```

Comment:
this was the thing which caused strange bundle folders. I changed the relative import to import an entire module (which is linked in the development) - this solved that